code/game/dna/genes/gene.dm

```diff
@@ -23,30 +23,47 @@
 	// Any of a number of GENE_ flags.
 	var/flags=0
 
+	// Trait to give
+	var/trait 
+
```

Comment:
Trait, activation messages & deactivation messages were moved up a level to gene.dm instead of gene/disability & gene/basic. The only gene that don't use it is monkey gene and frankly it's an overly snowflaked implementation (Seriously, messages are plural for power and singular for disabilities)